frontend/src/metabase/lib/urls/collections.ts

```diff
@@ -40,8 +41,7 @@ export function collection(collection?: Collection) {
     return `/collection/${id}`;
   }
 
-  const isPersonalCollection = typeof collection.personal_owner_id === "number";
-  const slug = isPersonalCollection
+  const slug = isRootPersonalCollection(collection)
```

Comment:
This was implemented exactly as this function's implementation

https://github.com/metabase/metabase/blob/a57007a03a6a9ba9172f950311fc6cb2828982b3/frontend/src/metabase/collections/utils.ts#L17-L21